### ISSUE TITLE
Fine-tuning : Check failed: crop_size == data_mean_.height()

### ISSUE THREAD

**Issue by cicido** [BVLC/caffe#6064]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Issue summary
where I run examples/02-fine-tuning.ipyb， there are erreos as follows:

F1121 15:29:51.496075 21338 data_transformer.cpp:258] Check failed: crop_size == data_mean_.height() (227 vs. 256) 
*** Check failure stack trace: ***
Aborted (core dumped)
I find in the function:
def style_net(train=True, learn_all=False, subset=None):
    if subset is None:
        subset = 'train' if train else 'test'
    source = caffe_root + 'data/flickr_style/%s.txt' % subset
    transform_param = dict(mirror=train, crop_size=227,
        mean_file=caffe_root + 'data/ilsvrc12/imagenet_mean.binaryproto')
    style_data, style_label = L.ImageData(
        transform_param=transform_param, source=source,
        batch_size=50, new_height=256, new_width=256, ntop=2)
    return caffenet(data=style_data, label=style_label, train=train,
                    num_classes=NUM_STYLE_LABELS,
                    classifier_name='fc8_flickr',
                    learn_all=learn_all)

### Your system configuration
Operating system: CentOS 7
Compiler: g++ (GCC) 4.8.5 20150623 (Red Hat 4.8.5-16)
CUDA version (if applicable):  8.0
CUDNN version (if applicable):6.0
BLAS: open
Python or MATLAB version (for pycaffe and matcaffe respectively):
Python 3.6.1 |Anaconda 4.4.0 (64-bit)| (default, May 11 2017, 13:09:58)

**Comment by cicido** [comment]
already solved.  
just comment the mean-file
https://github.com/BVLC/caffe/issues/4113
